src/usr/local/pfSense/include/www/alias-utils.inc

```diff
@@ -34,6 +34,16 @@ require_once("openvpn.inc");
 init_config_arr(array('aliases', 'alias'));
 $a_aliases = &$config['aliases']['alias'];
 
+$types = array(
```

Comment:
Rather than making a very generic name here, make it more specific, e.g. `$alias_types` or similar. That way it's more descriptive and less likely to conflict if something else happens to also be included that has its own `$types`.

Reply:
Got it. Patch updated.